.github/workflows/rspec-test.yml

```diff
@@ -67,10 +67,20 @@ jobs:
               npm run test
           fi
 
-      - name: Test
-        if: ${{ steps.secret-check.outputs.available != 'true' }}
+      - name: Run RSpec to Output File
+        id: rspec_test
         run: |
-          bundle exec rspec
+          bundle exec rspec --format json --out ${{ github.workspace }}/rspec_output.json
```

Comment:
Can we not read the output of the previous step, without having to explicitly output to a json file?

Reply:
I think we need to add `--format documentation` before `--format json` to still get our output (see https://til.hashrocket.com/posts/8d0e6c5ede-rspec-multiple-formats-with-different-out-buffers)

Reply:
research indicated the $GITHUB_OUTPUT was best used with variables and short strings and would not be best used with content from the logs

Reply:
Done, added new commits and screenshot.